### PR TITLE
[fix](nereids) select with specified partition name is not work as expected

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

This PR is to fix the select specific partition issue, certain codes related to this feature were accidentally deleted.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

File: regression-test/suites/nereids_p0/insert_overwrite_p0/large_wide_table.groovy
Comment (morrySnow) [inline regression-test/suites/nereids_p0/insert_overwrite_p0/large_wide_table.groovy:1]: remove this unrelated test

**Comment by morrySnow** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Pull request merged**